src/process/validation/util.ts

```diff
@@ -50,6 +50,9 @@ export function isObject(obj: any): obj is Object {
     return typeof obj != null && (typeof obj === 'object' || typeof obj === 'function');
 }
 
+const getCustomErrorMessage = ({ errorKeyOrMessage, context }: FieldError): string =>
```

Comment:
I'm not positive, but couldn't this be added to `getComponentErrorField`? the FieldError class already does a version of this, I'd like for this "error interpolation" stuff to all happen in one place